terraform/dev/main.tf

```diff
@@ -52,3 +52,17 @@ resource "cloudfoundry_service_instance" "database" {
   space        = data.cloudfoundry_space.space.id
   service_plan = data.cloudfoundry_service.rds.service_plans["micro-psql"]
 }
+
```

Comment:
I probably say this with most terraform I review but it seems like there should be _some_ way of reducing the duplication across all our environments.

Reply:
I can create modules, but then you have to call the module in every env and the only two lines would be deduplicated, one line if we need to change the service plan (bucket type) for any env.  Remember, this is configuration, not code.

Reply:
Also, given that this team is small and we will not be tinkering with these files ofter, I'm prioritizing readability over cleverness.